include/tvm/relay/attrs/transform.h

```diff
@@ -68,6 +68,21 @@ struct TakeAttrs : public tvm::AttrsNode<TakeAttrs> {
   }
 };
 
+struct SplitAttrs : public tvm::AttrsNode<SplitAttrs> {
+  Array<IndexExpr> indices_or_sections;
+  int axis;
+  bool equal_split;
+
+  TVM_DECLARE_ATTRS(SplitAttrs, "relay.attrs.SplitAttrs") {
+    TVM_ATTR_FIELD(indices_or_sections)
+        .describe("Number of outputs to be splitted");
+    TVM_ATTR_FIELD(axis).set_lower_bound(0).set_default(1)
```

Comment:
nit: numpy has axis defaults to 0, and does not seem to enforce axis to be nonnegative.

https://docs.scipy.org/doc/numpy/reference/generated/numpy.split.html